optimum/exporters/tasks.py

```diff
@@ -1148,7 +1198,21 @@ class TasksManager:
             onnx="VisionEncoderDecoderOnnxConfig",
         ),
         "vit": supported_tasks_mapping(
-            "feature-extraction", "image-classification", "masked-im", onnx="ViTOnnxConfig"
+            "feature-extraction",
+            "image-classification",
+            "masked-im",
+            onnx="ViTOnnxConfig",
+        ),
+        "vit-mae": supported_tasks_mapping(
+            "feature-extraction",
+            "masked-im",
```

Comment:
I think `"image-feature-extraction"` (or `"feature-extraction"`) should be enough https://huggingface.co/datasets/huggingface/transformers-metadata/blob/main/pipeline_tags.json#L1128
```suggestion
            "feature-extraction",
```

if not the correct class needed to load the model can be added to `_CUSTOM_CLASSES` : for vit-mae I only see `AutoModel` or `AutoModelForPreTraining`